tests/FrameCameraTests.cpp

```diff
@@ -73,31 +96,106 @@ TEST_F(FrameSensorModel, SlightlyOffCenter) {
 
 //Test all four corners:
 TEST_F(FrameSensorModel, OffBody1) {
-   csm::ImageCoord imagePt(14.5, -0.5);
+   csm::ImageCoord imagePt(15.0, 0.0);
    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
    EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
    EXPECT_NEAR(groundPt.y, -14.99325304, 1e-8);
    EXPECT_NEAR(groundPt.z, 14.99325304, 1e-8);
 }
 TEST_F(FrameSensorModel, OffBody2) {
-   csm::ImageCoord imagePt(-0.5, 14.5);
+   csm::ImageCoord imagePt(0.0, 15.0);
    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
    EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
    EXPECT_NEAR(groundPt.y, 14.99325304, 1e-8);
    EXPECT_NEAR(groundPt.z, -14.99325304, 1e-8);
 }
 TEST_F(FrameSensorModel, OffBody3) {
-   csm::ImageCoord imagePt(-0.5, -0.5);
+   csm::ImageCoord imagePt(0.0, 0.0);
    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
    EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
    EXPECT_NEAR(groundPt.y, 14.99325304, 1e-8);
    EXPECT_NEAR(groundPt.z, 14.99325304, 1e-8);
 }
 TEST_F(FrameSensorModel, OffBody4) {
-   csm::ImageCoord imagePt(14.5, 14.5);
+   csm::ImageCoord imagePt(15.0, 15.0);
    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
    EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
    EXPECT_NEAR(groundPt.y, -14.99325304, 1e-8);
    EXPECT_NEAR(groundPt.z, -14.99325304, 1e-8);
 }
 
+
+// Focal Length Tests:
+TEST_F(FrameIsdTest, FL500_OffBody4) {
+   isd.clearParams("focal_length");
+   isd.addParam("focal_length","500.0");
```

Comment:
This super nit picky. Can we use a single variable for the key? std::string key = "focal_length"? This will make refactoring later easier and is better form I think to help maintain these.